docs/api/spec.md

```diff
@@ -603,7 +613,7 @@ To find a User channel, one calls:
 ```ts
 // returns an array of channels
 const allChannels = await fdc3.getUserChannels();
-const redChannel = allChannels.find(c => c.id === 'red');
+const redChannel = allChannels.find(c => c.id === "red");
```

Comment:
I think this would be the correct example.
```js
const redChannel = allChannels.find(c => c.id === "fdc3.channel.1");
```

Reply:
Good point (match IDs to the recommended FDC3 channels). I'll implement that change.